frontend/src/components/DiscussionDetail/DiscussionDetailBottom.tsx

```diff
@@ -0,0 +1,52 @@
+import * as S from './DiscussionDetail.styled';
+import { ROUTES } from '@/constants/routes';
+import Modal from '@/components/common/Modal/Modal';
+import ConfirmButtons from '@/components/ModalContent/ConfirmButtons';
+import { useNavigate } from 'react-router-dom';
+import useModal from '@/hooks/useModal';
+import Button from '@/components/common/Button/Button';
+import { useDiscussionDelete } from '@/hooks/useDiscussionDelete';
+
+interface DiscussionDetailBottomProps {
+  discussionId: number;
+}
+
+export default function DiscussionDetailBottom({ discussionId }: DiscussionDetailBottomProps) {
+  const navigate = useNavigate();
+
+  const handleNavigateToUpdateDiscussion = () => {
+    navigate(`${ROUTES.submitDiscussion}?discussionId=${discussionId}`);
+  };
+
+  const { isModalOpen, handleModalClose, handleModalOpen } = useModal();
+  const { discussionDeleteMutation } = useDiscussionDelete();
+
+  const handleDiscussionDeleteCallback = () => {
+    handleModalClose();
+    navigate(ROUTES.discussions);
+  };
+
+  const handleDiscussionDelete = () => {
+    discussionDeleteMutation(discussionId, handleDiscussionDeleteCallback);
+  };
+
+  return (
+    <>
+      <S.DiscussionDetailBottom>
+        <Button variant="defaultText" onClick={() => handleNavigateToUpdateDiscussion()}>
```

Comment:
```suggestion
        <Button variant="defaultText" onClick={handleNavigateToUpdateDiscussion}>
```

아주 사소하지만,, 이렇게 바꿔도 좋을 것 같아요!

Reply:
오오 발견해주셔서 감사해요👍🏻